web/httpserver.py

```diff
@@ -5,23 +5,10 @@
 from . import utils
 from . import webapi as web
 
-try:
-    from io import BytesIO
-except ImportError:
-    from StringIO import BytesIO
-
-try:
-    from http.server import HTTPServer, SimpleHTTPRequestHandler, BaseHTTPRequestHandler
-except ImportError:
-    from SimpleHTTPServer import SimpleHTTPRequestHandler
-    from BaseHTTPServer import HTTPServer, BaseHTTPRequestHandler
-
-try:
-    from urllib import parse as urlparse
-    from urllib.parse import unquote
-except ImportError:
-    import urlparse
-    from urllib import unquote
+from io import BytesIO
+from http.server import HTTPServer, SimpleHTTPRequestHandler, BaseHTTPRequestHandler
```

Comment:
```suggestion
from http.server import BaseHTTPRequestHandler, HTTPServer, SimpleHTTPRequestHandler
```
Like `isort` would do.